src/main/java/org/gatorgradle/display/CheckResult.java

```diff
@@ -137,7 +137,7 @@ public String textReport(boolean includeDiagnostic) {
     if (outcome) {
       return PASS_SYMBOL + INDENT + check;
     } else if (check.contains("--reach")) {
-      outcome = True;
+      outcome = outcome;
```

Comment:
This is a no-op; it should probably just not be there, if we don't want to affect the value of outcome.

Reply:
This was changed in the most recent commit and no longer exists in the code. Thank you.

Reply:
@schultzh sorry about that, I guess I didn't have the "review all commits" fully selected!